dev-resources/template/1_db.yml

```diff
@@ -1,12 +1,14 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: "Postgres DB for SQL LRS"
 Parameters:
-  # Networking
+  VPCStackName:
+    Description: Name of VPC Stack
```

Comment:
Might want to specify something like (Optional, but if not provided must provide VPCId and DBSubnets).

Reply:
updated doc string via https://github.com/yetanalytics/lrsql/pull/276/commits/c8065160f73210f44faa63d563556f479cac46bb